app/answers.csv

```diff
@@ -0,0 +1,1375 @@
+"Question text","Answer type","Required","Image port","Answer","Images","Creator"
+"Measure retrosternal space","Distance measurement","False","Secondary port,","{'end': [131.34259033203125, 125.7043685913086, 0.0], 'name': '', 'type': 'Distance measurement', 'start': [128.30662536621094, 126.1380844116211, 0.0], 'version': {'major': 1, 'minor': 0}}","1.3.12.2.1107.5.4.4.1163.30000011060706174378100000096.mha; 1.3.12.2.1107.5.4.4.1163.30000011060706174378100000100.mha","BramVanGinneken"
+"Lat - #4 Flattening of diaphragmatic contours","Bool","False","","False","1.3.12.2.1107.5.4.4.1163.30000011060706174378100000096.mha; 1.3.12.2.1107.5.4.4.1163.30000011060706174378100000100.mha","BramVanGinneken"
+"PA - #1 Flattening of the diaphragm","Bool","False","","False","1.3.12.2.1107.5.4.4.1163.30000011060706174378100000096.mha; 1.3.12.2.1107.5.4.4.1163.30000011060706174378100000100.mha","BramVanGinneken"

```

Comment:
I think that this file was committed by accident, please remove it

Reply:
Indeed, sorry about that.

Reply:
It's ok, this is why we review :-)